realm/realm-library/src/main/java/io/realm/internal/TableView.java

```diff
@@ -510,8 +510,9 @@ public long upperBoundLong(long columnIndex, long value) {
     public TableView findAllLong(long columnIndex, long value){
         // Execute the disposal of abandoned realm objects each time a new realm object is created
         context.executeDelayedDisposal();
-        long nativeViewPtr = nativeFindAllInt(nativePtr, columnIndex, value);
-        try { 
+        long nativeViewPtr = 0;
+        try {
+            nativeViewPtr = nativeFindAllInt(nativePtr, columnIndex, value);
```

Comment:
I agree, I am not sure that these changes will prevent anything. 
Also, if a change isn't accompanied by unit tests, please add a description explaining the rationale as we have to be extra careful reviewing those changes.